bench/wtperf/wtperf_run_py/wtperf_run.py

```diff
@@ -193,15 +196,34 @@ def setup_perf_stats():
                                     output_label='Min update throughput'))
     perf_stats.add_stat(PerfStatCount(short_label="warnings",
                                       pattern='WARN',
-                                      output_label='Warnings'))
-    perf_stats.add_stat(PerfStatLatency(short_label="max_latencies",
+                                      output_label='Latency warnings'))
+    perf_stats.add_stat(PerfStatLatency(short_label="top5_latencies_read_update",
                                         stat_file='monitor.json',
-                                        output_label='Latency Max',
+                                        output_label='Latency(read, update) Max',
+                                        ops = ['read', 'update'],
                                         num_max = 5))
     perf_stats.add_stat(PerfStatCount(short_label="eviction_page_seen",
                                       stat_file='WiredTigerStat*',
                                       pattern='[0-9].wt cache: pages seen by eviction',
                                       output_label='Pages seen by eviction'))
+    perf_stats.add_stat(PerfStatLatency(short_label="max_latency_insert",
+                                        stat_file='monitor.json',
+                                        output_label='Latency(insert) Max',
+                                        ops = ['insert'],
+                                        num_max = 1))
+    perf_stats.add_stat(PerfStatLatency(short_label="max_latency_read_update",
+                                        stat_file='monitor.json',
+                                        output_label='Latency(read, update) Max',
+                                        ops = ['read', 'update'],
+                                        num_max = 1))
+    perf_stats.add_stat(PerfStatMax(short_label="max_read_throughput",
+                                    pattern=r'updates,',
```

Comment:
Nitpick: Change this pattern to `r'reads,'`. Same for `min_read_throughput` below

Reply:
I did that before, but it also picked up `threads, ` might have to some other way, perhaps `r'\d+ reads,'`

Reply:
No worries then, happy to keep `updates,`